provisioning/local/main.tf

```diff
@@ -0,0 +1,39 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.11"
+    }
+
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.68"
+    }
+
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = "~> 2.41"
+    }
+
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 4.74.0"
+    }
+  }
+
+  backend "s3" {
+    role_arn = "arn:aws:iam::681277395786:role/kbc-local-dev-terraform"
+    region = "eu-central-1"
+    bucket = "local-dev-terraform-bucket"
+    dynamodb_table = "local-dev-terraform-table"
+  }
+}
+
+locals {
+  app_name = "job-runner"
+  app_display_name = "Job Runner"
```

Comment:
Tohle je cool, na většině míst to app_name máme natvrdo na x místech. Asi by bylo dobrý to přidat do conflu do tvého návodu nebo do best practices pro terraform :)